pyk/src/pyk/ktool/krun.py

```diff
@@ -0,0 +1,56 @@
+import json
+import logging
+from pathlib import Path
+from subprocess import CalledProcessError, CompletedProcess
+from tempfile import NamedTemporaryFile
+from typing import Final, List, Optional
+
+from ..cli_utils import check_file_path, run_process
+from ..cterm import CTerm
+from ..kast import KAst, KInner
+from .kprint import KPrint
+
+_LOGGER: Final = logging.getLogger(__name__)
+
+
+def _krun(
+    definition_dir: Path,
+    input_file: Path,
+    depth: Optional[int] = None,
+    args: List[str] = [],
```

Comment:
Rename this `additional_args` for consistency with `kompile`.

Reply:
I'm actually renaming `kompile` to use `args` directly instead, which is more consistent with the other tools (`krun`, `kprove`, and `kprint`).

Reply:
I'm going to go ahead and merge this, so I can open my next PR. Let me know if you want to see this switched back.

Reply:
Agh, I re-requested your review :P Not sure github will let me merge, but I'll leave automerge on for now.